samples/nrf9160/modem_shell/src/th/th_shell.c

```diff
@@ -79,10 +89,13 @@ static int cmd_th(const struct shell *shell, size_t argc, char **argv)
 
 SHELL_STATIC_SUBCMD_SET_CREATE(
 	sub_bg,
-	SHELL_CMD(startbg, NULL, "iperf3 | ping <params>\nStart a background thread.\n",
+	SHELL_CMD(startbg, NULL, "iperf3 | ping <params>\nStart a background thread.",
 		  cmd_th_startbg),
 	SHELL_CMD(startfg, NULL, "iperf3 | ping <params>\nStart a foreground thread.",
 		  cmd_th_startfg),
+	SHELL_CMD(pipeline, NULL, "\"<cmd> <params>\" \"<cmd> <params>\" \"<cmd> <params>\"...\n"
+				  "Execute any MoSh commands sequentially in one thread.",
+		  cmd_th_start_pipeline),
```

Comment:
I'm tempted to use `SHELL_CMD_ARG` here also but I'm not sure how it works with unknown number of optional arguments. I know we are adding a big enough number there in some cases but this would require some testing. Maybe just easier to use `SHELL_CMD` although we are lacking the automatic check for too few parameters.

Reply:
Thanks for review. I implemented other proposed changes but this as I think there is a case for using SHELL_CMD.